README.md

```diff
@@ -1,34 +1,23 @@
 # linter-htmlhint
 
-This linter plugin for [Linter](https://github.com/AtomLinter/Linter) provides
-an interface to [htmlhint](https://github.com/yaniswang/HTMLHint). It will be
+A plugin for [Linter](https://github.com/atom-community/linter) providing an interface to [HTMLHint]. It will be
 used with files that have the syntax.
 
 ## Installation
 
-Linter package must be installed in order to use this plugin. If Linter is not
-installed, please follow the instructions [here](https://github.com/AtomLinter/Linter).
-
-### Plugin installation
+Linter package must be installed in order to use this plugin. If Linter is not installed, please follow the instructions [Linter].
```

Comment:
If you want an example of what I was talking about, [here](https://github.com/AtomLinter/linter-bootlint/blob/master/README.md#installation) is one that I recently wrote.

What you have here will work though if you change the last part to:
>If [Linter] is not installed, it will be installed for you.